### DISCUSSION TITLE
3,5" IPS ILI9488 on Arduino Due with hardware SPI

### DISCUSSION THREAD

**Discussion by PatMan6889** [moononournation/Arduino_GFX#573] [Q&A]
Hello!

I got an ILI9488 3,5" LCD as IPS version and want to run it on an Arduino Due.

I use this software SPI initialization and it is working fine:

```
Arduino_DataBus *bus = new Arduino_SWSPI(TFT_DC /* DC */, TFT_CS /* CS */, TFT_SCK /* SCK */, TFT_MOSI /* MOSI */, -1 /* MISO */);

Arduino_GFX *gfx = new Arduino_ILI9488_18bit(bus, TFT_RST /* RST */, 3 /* rotation */, true/* IPS */);
```
I would like to use hardware SPI but I cannot get it to run. All I see is a blank white screen:
`Arduino_DataBus *bus = new Arduino_HWSPI(TFT_DC /* DC */, TFT_CS /* CS */);`

Other SPI devices work with the same wiring.
Do you have any idea what is wrong?

Best regards

**Comment by moononournation** [comment]
try lowering the speed

**Reply by PatMan6889** [reply]
> try lowering the speed

I tried with 
```
bus->begin(8000000);
gfx->begin(8000000);
```
to lower the speed but no success. I also went down to 1 MHz.

**Comment by moononournation** [comment]
then no idea, I don not have Arduino Due in hand so cannot support it
